README.md

```diff
@@ -56,12 +56,11 @@ def handler(event, context):
 The following may be set as kwargs to the IOpipe class initializer:
 
 - client_id: your client_id (register at [www.iopipe.com](https://www.iopipe.com)
-- debug: enable debug logging.
-- url: custom URL for reporting metrics
+- debug: enable debug logging by setting this to `True`
```

Comment:
did we kill 'url' or did we not support it? (we should support configurable urls.. it's useful for our own testing)